faq/billing.mdx

```diff
@@ -111,6 +111,12 @@ If after several manual attempts you cannot manage to trigger a successful trans
   If you are unable to settle your invoice after the due date, your account is going to be suspended and your resources will be unreachable. You will then have another ten days to regularize your invoice before your resources are deleted.
 </Message>
 
+## How long do I have to pay my invoice before my resources get locked?
+After your invoice is issued, you have ten days to pay it. If you fail to settle your invoice within this period, your account will be suspended, and your resources will become inaccessible. You will have another ten days to regularize your invoice before your resources are deleted.
```

Comment:
I think "regularize" an invoice is French, and not really used in English. I see it's on multiple places on this page - I'd change them all to "pay your invoice" or "settle your invoice".